designs/administrator/4.5/alertmanager-config/index.md

```diff
@@ -0,0 +1,26 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Alertmanager Configuration
+
+![](img/alertmanager-first-time.png)
+- The default alertmanager configuration will ship with 3 receivers, two of which need to be configured to correctly function.
+
+![](img/alertmanager-editing-default.png)
+- The default receiver will continue to display an informational alert when it is configured explaining its purpose.
+
+![](img/alertmanager-incomplete-critical.png)
+- When only one receiver remains unconfigured, the text of the notification should change from plural to singular.
+
+![](img/alertmanager-editing-critical.png)
```

Comment:
More nits: I think the first sentence is supposed to be "The routing labels..." instead of "This routing labels". Also probably not a huge deal but curious if it's weird to say you can delete this thing but there's no way to do so from the page you're looking at – I kind of expected to be able to after reading that.

Reply:
Thanks for the fix. I ended up changing the wording here a bit that hopefully makes that less weird